libdeno/internal.h

```diff
@@ -39,25 +78,51 @@ struct InternalFieldData {
 void Print(const v8::FunctionCallbackInfo<v8::Value>& args);
 void Recv(const v8::FunctionCallbackInfo<v8::Value>& args);
 void Send(const v8::FunctionCallbackInfo<v8::Value>& args);
+void Shared(const v8::FunctionCallbackInfo<v8::Value>& args);
 void SetGlobalErrorHandler(const v8::FunctionCallbackInfo<v8::Value>& args);
 void SetPromiseRejectHandler(const v8::FunctionCallbackInfo<v8::Value>& args);
 void SetPromiseErrorExaminer(const v8::FunctionCallbackInfo<v8::Value>& args);
 static intptr_t external_references[] = {
     reinterpret_cast<intptr_t>(Print),
     reinterpret_cast<intptr_t>(Recv),
     reinterpret_cast<intptr_t>(Send),
+    reinterpret_cast<intptr_t>(Shared),
```

Comment:
@piscisaureus Replying to your comment in another thread - I don't know what to put here if libdeno.shared() is a constant ArrayBuffer. Any idea?

Reply:
You put the base address of the buffer. So buf.alloc_ptr.

Reply:
Fixed. `libdeno.shared` is a getter now. I cannot use `buf.data_ptr` here directly because `external_references` is statically allocated.